<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DecodificadorDlg</class>
 <widget class="QDialog" name="DecodificadorDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>194</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Decodificador</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Valor de la Resistencia:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblResistencia">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="KColorCombo" name="kColorBanda1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>61</width>
     <height>38</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string notr="true"/>
   </property>
   <property name="color">
    <color>
     <red>0</red>
     <green>0</green>
     <blue>0</blue>
    </color>
   </property>
  </widget>
  <widget class="KColorCombo" name="kColorBanda2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>61</width>
     <height>38</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string notr="true"/>
   </property>
   <property name="color">
    <color>
     <red>0</red>
     <green>0</green>
     <blue>0</blue>
    </color>
   </property>
  </widget>
  <widget class="KColorCombo" name="kColorBanda3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>61</width>
     <height>38</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string notr="true"/>
   </property>
   <property name="color">
    <color>
     <red>0</red>
     <green>0</green>
     <blue>0</blue>
    </color>
   </property>
  </widget>
  <widget class="KColorCombo" name="kColorBanda4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>61</width>
     <height>38</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string notr="true"/>
   </property>
   <property name="color">
    <color>
     <red>0</red>
     <green>0</green>
     <blue>0</blue>
    </color>
   </property>
  </widget>
  <widget class="KColorCombo" name="kColorBanda5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>61</width>
     <height>38</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string notr="true"/>
   </property>
   <property name="color">
    <color>
     <red>0</red>
     <green>0</green>
     <blue>0</blue>
    </color>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb4Bandas">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>0</y>
     <width>110</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>4 Bandas</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb5Bandas">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>0</y>
     <width>110</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>5 Bandas</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDecodificar">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>90</y>
     <width>93</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Decodificar</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblResistencia_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Tolerancia</string>
   </property>
  </widget>
  <zorder>lblResistencia_2</zorder>
  <zorder>label</zorder>
  <zorder>lblResistencia</zorder>
  <zorder>kColorBanda1</zorder>
  <zorder>kColorBanda2</zorder>
  <zorder>kColorBanda3</zorder>
  <zorder>kColorBanda4</zorder>
  <zorder>kColorBanda5</zorder>
  <zorder>rb4Bandas</zorder>
  <zorder>rb5Bandas</zorder>
  <zorder>btnDecodificar</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorCombo</class>
   <extends>QComboBox</extends>
   <header>kcolorcombo.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
